zkevm-circuits/src/table.rs

```diff
@@ -177,6 +175,12 @@ pub enum TxFieldTag {
     TxHash,
     /// TxType: Type of the transaction
     TxType,
+    /// Access list address count (EIP 2930)
+    AccessListAddressesLen,
+    /// Access list all storage key count (EIP 2930)
+    AccessListStorageKeysLen,
+    /// Gas cost for access list (EIP 2930)
+    AccessListGasCost,
```

Comment:
We will also need a `AccessListRlc` field type. However I am not sure if @darth-cy is adding it as a part of his PR. If he is not, it should be added here.

Reply:
Add `AccessListRLC` in commit cd6ac3a. May need @darth-cy help check if [the RLC calculation](https://github.com/scroll-tech/zkevm-circuits/pull/1045/files#diff-864796c9d6f832182c9e2c72268681463f4af9eaac5cf56b04f56b3a614923fbR1972-R1980) is correct when implements constraints in tx-circuit.